source/release-notes/backend.txt

```diff
@@ -13,6 +13,18 @@ Atlas App Services Changelog
 2022 Releases
 -------------
 
+.. _backend_20221214:
+
+14 December 2022 Release
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: waiting on link for Admin API endpoints for changing expiration times. will add when ready. 
+
+- Raise :ref:`Atlas Function request timeout <function-constraints>`
+  from 150 seconds to 180 seconds.
+- Let users configure expiration times for refresh tokens from the Admin API.
```

Comment:
instead of using language like 'Let users' can we just say 'Expiration times for refresh tokens can be configured from the API'

Reply:
i get how this sounds funny. done b/c we try to avoid passive voice in the docs. i'll use your phrasing.

Reply:
```suggestion
- Expiration times for refresh tokens can be configured from the API.
```